pkg/image/imageversion.go

```diff
@@ -36,8 +39,12 @@ var (
 const (
 	// ConformanceImageVersionAuto represents detecting the server's kubernetes version.
 	ConformanceImageVersionAuto = "auto"
-	// ConformanceImageVersionLatest represents always using the server's latest version.
+	// ConformanceImageVersionLatest represents always using the server's latest dev version.
 	ConformanceImageVersionLatest = "latest"
+
+	// DevVersionURL is the URL which should respond with a simple text of the latest version for devs.
+	DevVersionURL      = "https://storage.googleapis.com/k8s-release-dev/ci/latest.txt"
+	DevVersionImageURL = "gcr.io/k8s-staging-ci-images/conformance"
```

Comment:
oh wait isn't this supposed to be a `k8s.gcr.io` URL?

Reply:
nope. this is the image minted by CI jobs against tip of master. we don't push those to prod registry